### PR TITLE
random test to be modified later

### PR DESCRIPTION
**Pull request by midizzz**

### CODE REVIEW

File: content/vault/.obsidian/workspace.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
             "state": {
               "type": "markdown",
               "state": {
-                "file": "SatoshiPlace.md",
+                "file": "BlogPost1.md",
```
Comment (dragon-foobar) [inline content/vault/.obsidian/workspace.json]: Not sure why you need to do this?

File: content/vault/BlogPost1.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  - welcome
+  - newbie
+  - open source
+  - franchise
+
+---
+
+# Welcome!
+
+
```
Comment (dragon-foobar) [inline content/vault/BlogPost1.md]: I like what you've done here, but maybe you could customise the blog to make it unique ?

**Review by dragon-foobar — CHANGES_REQUESTED** [review]
hmm.. good first try dude. Maybe you could write an article that isn't the same as welcome.

File: content/vault/BlogPost1.md [hunk 3]
```diff
@@ -9,21 +9,58 @@ tags:
 
 ---
 
-# Welcome!
+# MultiSig Magic Trust Machine
```
Comment (unspentspace-admin) [inline content/vault/BlogPost1.md]: Nice title!

**Comment by unspentspace-admin** [comment]
Awesome, man. Just a few spelling/grammar things. Maybe easier if I add a commit on? @midizzz

**Comment by midizzz** [comment]
> Awesome, man. Just a few spelling/grammar things. Maybe easier if I add a commit on? @midizzz 

Yeah. I want to submit again. Then you can proof it. Still to ruff. Just testing the method. Seems it works

**Comment by midizzz** [comment]
ok, not sure if I am typing in the right place.. It looks like you have seen this, maybe there's somthing about a tag "satoshis place"? no idea. There must be some typos etc?

**Comment by dragon-foobar** [comment]
@midizzz there are a couple of typos still, but that's not the main issue. The main issue is that your branch is now not in harmony with the main branch. You need to merge dev branch into your branch. We can do a screen share or something.

**Comment by midizzz** [comment]
tbh at the moment if feels like the communication here falls into black hole

**Comment by nickfarrow** [comment]
Rebased and removed excess changes (DS_STORE and obsidian changes)

I also did a proof read, with a solid amount of wording changes and grammar fixes. Inspiring post, I want to start a peer multisig now. Please check I have accurately captured your words 0b117ce4a05ab624553914c996de2242165877a4

I suggest to hit "Squash and Merge" for this one so we can keep the commits clean

**Comment by midizzz** [comment]
Thanks @nickfarrow .. Oh no. I have also reviewed this cutting it down a lot and have been trying to resubmit. I am just getting used to the Obsidian to Github flow here.. I better have a look at your edits in context of the new revision named Magic MultiSig Trust Machine.

**Pull request closed without merging**